apps/site/assets/js/support-form.js

```diff
@@ -407,3 +408,9 @@ function showExpandedForm($) {
 function hideExpandedForm($) {
   $(".support-form-expanded").hide();
 }
+
+function focusOnContentWhenExpanding($) {
+  $("div[id^='panel-']").on("shown.bs.collapse", function() {
+    $(this).focus();
+  });
+}
```

Comment:
Did you want to try making use of the event listener that uses a `.js-focus-on-expand` class? Looks like it works the same as you did here. https://github.com/mbta/dotcom/blob/eaac9fdb9b32f0c55f64689ec23de01de283a513/apps/site/assets/js/collapse.js#L20-L21

Reply:
Arg, I didn't think to check that. Agreed, no need to duplicate.